odoo/modules/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+# ruff: noqa: F401
+# Exposed here so that exist code is unaffected.
+from odoo.orm.models import MetaModel
```

Comment:
You don't need to expose it here, you can import from `odoo.orm` directly in `odoo.modules`.